src/Controls/src/Core/CompatServiceProvider.cs

```diff
@@ -12,6 +12,7 @@ static class CompatServiceProvider
 		static IServiceProvider? _serviceProvider;
 		static IEmbeddedFontLoader? _embeddedFontLoader;
 
+		// TODO MAUI This create seems wrong
```

Comment:
Which it probably is but I think this whole class will just get deleted once we get essentials and CG maui-ified